### PR TITLE
Lakshya

### PR DESCRIPTION
**Pull request by king-11**
- Navbar and Footer Added
- Instructor view refactored
  - Row and cols instead of layout and flex
  - toolbar sizing and shouldn't overlap with main app bar

> if all good perform a rebase and merge

### CODE REVIEW

File: components/Navbar.vue [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+        </v-list-item>
+      </v-list>
+      <v-spacer />
+    </v-navigation-drawer>
+    <v-app-bar light elevate-on-scroll app>
+      <v-app-bar-nav-icon x-large class="black--text text--darken-5" @click="navbar = !navbar" />
+      <v-toolbar-title class="text-uppercase">
+        <div style="display:inline-block;transform: rotateY(180deg)">
+          P
+        </div>2P
```
Comment (MadhavaDasari) [inline components/Navbar.vue:47]: creative 😉

File: pages/instructor.vue [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <template>
-  <div>
-    <v-toolbar app>
+  <v-container fluid>
+    <v-toolbar max-width="800" class="mx-auto">
```
Comment (MadhavaDasari) [inline pages/instructor.vue:3]: I feel  the previous version itself looked good i.e both navbar and toolbar sticked together and toolbar occupied the whole page  .. just have a look at it

**Review by MadhavaDasari — CHANGES_REQUESTED** [review]
everything else looks good  for the initial stage 👍 . ( hope there will be changes in the items of navbar and navigation-drawer in future )

File: pages/instructor.vue (hunk 2 above)
Comment (king-11) [reply pages/instructor.vue:3]: on large screen it looked a bit odd I guess and for small screens it was cool. I will add a breakpoint for md screens

Comment (king-11) [reply pages/instructor.vue:3]: I will add an issue you can then resolve it for toolbar.

File: pages/instructor.vue [hunk 3]
```diff
@@ -1,14 +1,14 @@
 <template>
   <v-container fluid>
-    <v-toolbar max-width="800" class="mx-auto">
+    <v-toolbar class="mx-xs-0 mx-sm-4 mx-md-10 mx-lg-14 mx-xl-16 rounded-lg">
```
Comment (MadhavaDasari) [inline pages/instructor.vue:3]: perfect 👍

File: pages/instructor.vue [hunk 4]
```diff
@@ -1,14 +1,14 @@
...
-        <h2>Your courses</h2>
+        <strong>Your Courses</strong>
       </v-toolbar-title>
       <v-spacer />
       <v-tooltip bottom>
         <template v-slot:activator="{ on, attrs }">
-          <v-btn class="white black--text" v-bind="attrs" v-on="on">
-            Create Course
+          <v-btn class="white black--text" v-bind="attrs" fab x-small v-on="on">
+            <v-icon>mdi-plus</v-icon>
```
Comment (MadhavaDasari) [inline pages/instructor.vue:11]: felt the previous was a bit user friendly ( as it was key button in the page .. ) , instead lets remove that tool-tip and keep the button as it is . Even this is fine,  am Merging it let's look at it later 👍

File: pages/instructor.vue (hunk 3 above)
Comment (MadhavaDasari) [inline pages/instructor.vue:3]: perfect 👍

**Pull request merged**